app/assets/stylesheets/responsive.scss

```diff
@@ -29,6 +29,48 @@
     padding: 10px 10px 0 10px;
     border-bottom: 0;
   }
+
+  // Responsive table behvavior inspired by
+  // https://css-tricks.com/responsive-data-tables/
+  .js--responsive_table {
```

Comment:
I don't have a strong feeling on this, but would the current best practices say to use separate classes for the styles and the JS? (this project is an amalgam of best practices from various points in time).